### ISSUE TITLE
Spindle Coolant Unexpected UI Behaviour 

### ISSUE THREAD

**Issue by AustinSaintAubin** [cncjs/cncjs#142]
The Coolant Buttons in the Spindle widget do not behave as expected.
I have two separate cooling / chip removal systems.
One is connected to GRBL's Flood Coolant, the other is connected to the GRBL's Mist Coolant.
I am able to enable/disable them independently or have them both on, which is good, I want the ability to have both on/off/or just one on.

<img width="329" alt="screen shot 2017-03-01 at 11 07 31 pm" src="https://cloud.githubusercontent.com/assets/969780/23493939/1ca3d30a-fed5-11e6-9c3c-691ef83b6b5e.png">

**The issue is that the UI buttons do not reflect the state of the Flood Coolant, and Mist Coolant properly**.
If I enable Flood Coolant, then also enable Mist Coolant, both show on the UI as off.
They Flood Coolant and Mist Coolant should be able to be toggled independently of each other in the UI.

[cncjs@v1.9.0-alpha.4]

**Comment by cheton** [comment]
@AustinSaintAubin 

May you show me the result of your $G command?

I just traced the code at https://github.com/gnea/grbl/blob/master/grbl/report.c#L321, I guess it may report M78 when both mist and flood are on.
```c
 report_util_gcode_modes_M();
  #ifdef ENABLE_M7
    if (gc_state.modal.coolant) { // Note: Multiple coolant states may be active at the same time.
      if (gc_state.modal.coolant & PL_COND_FLAG_COOLANT_MIST) { serial_write('7'); }
      if (gc_state.modal.coolant & PL_COND_FLAG_COOLANT_FLOOD) { serial_write('8'); }
    } else { serial_write('9'); }
  #else
    if (gc_state.modal.coolant) { serial_write('8'); }
    else { serial_write('9'); }
  #endif
```

**Comment by cheton** [comment]
Hi @chamnit

I uncommented ENABLE_M7 from config.h and found something strange. The $G parser state reported <b>M78</b> when both M7 (Mist) and M8 (Flood) are turned on, may I check if this is a desired behavior? Thanks.

![image](https://cloud.githubusercontent.com/assets/447801/23515967/c1fe6dac-ffa6-11e6-8998-0cb51b1e449b.png)

**Comment by chamnit** [comment]
@cheton : It should say `M7 M8`. Thanks for letting me know. I'll fix it asap.

**Comment by cheton** [comment]
It looks great. Thank you for your prompt assistance.

https://github.com/gnea/grbl/commit/849122559b647493f4deda59405f130589fcd1b6

**Comment by AustinSaintAubin** [comment]
Awesome!

**Comment by AustinSaintAubin** [comment]
CNCjs UI still needs work, if both M7 & M8 are on UI shows only M8 is on.
But output looks good.

```
> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M9 T0 F0 S40
ok
> M7
ok
> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M7 T0 F0 S40]
ok
> M8
ok
> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M7 M8 T0 F0 S40]
ok
> M9
ok
> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M9 T0 F0 S40]
ok
> M8
ok
> $G
[GC:G0 G54 G17 G21 G90 G94 M5 M8 T0 F0 S40]
ok

**Comment by cheton** [comment]
Fixed in 1.9.0-alpha.5
